### PR TITLE
Qstatus

### PR DESCRIPTION
**Pull request by ibmmqmet**
Please ensure all items are complete before opening.

- [x] Tick to sign-off your agreement to the [IBM Contributor License Agreement](https://github.com/ibm-messaging/mq-golang/CLA.md)
- [ ] You have added tests for any code changes
- [x] You have updated the [CHANGES.md](https://github.com/ibm-messaging/mq-golang/CHANGES.md)
- [ ] You have completed the PR template below:

## What

Add optional DIS QSTATUS metrics - which can be used on z/OS for minimal information

## How

Add a queue.go following the pattern from channel.go, but had to modify names for uniqueness in package. Now use extended PCF command (_XR) format for z/OS compatibilty.

## Testing

Used sample dashboards to collect data

### CODE REVIEW

File: ibmmq/mqiMQCNO.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ void setCCDTUrl(MQCNO *mqcno, PMQCHAR url, MQLONG length) {
 	mqcno->CCDTUrlOffset = 0;
 	mqcno->CCDTUrlPtr = NULL;
 	mqcno->CCDTUrlLength = length;
-	if (url != NULL) {
+	if (url != NULL && length > 0) {
```
Comment (parrobe) [inline ibmmq/mqiMQCNO.go]: Copyright date for this file needs to update.

File: ibmmq/mqiPCF.go [hunk 2]
```diff
@@ -252,6 +252,15 @@ func ReadPCFParameter(buf []byte) (*PCFParameter, int) {
 	case C.MQCFT_GROUP:
 		binary.Read(p, endian, &pcfParm.Parameter)
 		binary.Read(p, endian, &pcfParm.ParameterCount)
+
```
Comment (parrobe) [inline ibmmq/mqiPCF.go]: Copyright for this file needs updating

File: mqmetric/queue.go [hunk 3]
```diff
@@ -0,0 +1,351 @@
+/*
+Package mqmetric contains a set of routines common to several
+commands used to export MQ metrics to different backend
+storage mechanisms including Prometheus and InfluxDB.
+*/
+package mqmetric
+
+/*
+  Copyright (c) IBM Corporation 2016, 2018
```
Comment (parrobe) [inline mqmetric/queue.go]: If this is a new file the copyright start date should be 2018 not 2016

Comment (ibmmqmet) [reply mqmetric/queue.go]: Done

**Pull request merged**